docs/sources/flow/release-notes.md

```diff
@@ -41,11 +41,157 @@ This functionality is now only available in the main configuration.
 The `disable_high_cardinality_metrics` configuration argument is used by `otelcol.exporter` components such as `otelcol.exporter.otlp`.
 If you need to see high cardinality metrics containing labels such as IP addresses and port numbers, you now have to explicitly set `disable_high_cardinality_metrics` to `false`.
 
+### Breaking change: Label for children blocks of `prometheus.exporter.cloudwatch`, `prometheus.exporter.snmp` and `pyroscope.scrape` are removed.
+
+Previously in aforementioned components, there are children blocks that requires labels which would be used for naming. 
+More information in the following table.
```

Comment:
```suggestion
Now, you must specify a `name` attribute for the following child blocks.
```

What about moving the text from line 55 up to before the table and describing it this way? I dropped the "less restrictive" part. It's not really adding a lot... unless it's a really important point.